packages/wallet/src/redux/reducers/__tests__/closing.test.ts

```diff
@@ -2,7 +2,7 @@ import { walletReducer } from '..';
 
 import * as states from '../../../states';
 import * as actions from '../../actions';
-import * as outgoing from 'wallet-client/lib/interface/from-wallet';
+import * as outgoing from '../../../../../wallet-client/lib/interface/from-wallet';
```

Comment:
@geoknee Do you know why imports with the relative path are not working?

Reply:
I do not know. What I do know is that somehow wallet-client found its way into `package.json`, and subsequently we were trying to import from a completely unrelated node module.

It *could* be that with that dependency removed and rerunning `yarn install`, the relative path imports may start working again. I will check.

Reply:
Turns out I hadn't followed the instructions in the monorepo readme. It seems `npx lerna bootstrap` populates the `node_modules` in the wallet package with "our" `wallet-client`, allowing it to be imported.

Will revert to absolute paths (i.e. without `../../`) .

Reply:
^ It will only do so, however, if `wallet-client` is listed in `package.json`. 

To get it to work, I had to run `yarn install` in the tictactoe directory, and *then* `npx lerna bootstrap` in the top directory, which presumably overwrites the unrelated external package with our `client-wallet` folder. This seems a bit inelegant.